components/ServiceList/index.js

```diff
@@ -24,18 +26,19 @@ export default function ServiceList() {
   return (
     <>
       <h2>Services</h2>
+      <div>
+        <Link href="/create">+ add service</Link>
+      </div>
```

Comment:
You probably do not need the `div` element here.
```suggestion
        <Link href="/create">+ add service</Link>
```